graylog2-web-interface/src/components/common/SourceCodeEditor.jsx

```diff
@@ -15,6 +15,8 @@ import 'brace/mode/text';
 import 'brace/mode/yaml';
 import 'brace/theme/tomorrow';
 import 'brace/theme/monokai';
+
+import PipelineRulesMode from './mode-pipeline.js';
```

Comment:
Running a production build with the change I am seeing this error on the console:

`Uncaught SyntaxError: Unexpected token <` on `mode-pipeline.js:1`

Looking a bit more, I am seeing a network request to `/system/pipelines/rules/mode-pipeline.js`, which does not match anything on the server, so it returns the web interface's HTML document.

I have the suspicion that this is caused because `SourceCodeEditor` is loaded through `loadable-components`, but I'm not 100% sure about it.

Reply:
Anyway, the highlighting appears to work even with the console error, and makes it nicer to read the code! 🎉